safeeyes/core.py

```diff
@@ -16,32 +16,25 @@
 
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-"""
-SafeEyesCore provides the core functionalities of Safe Eyes.
-"""
+"""SafeEyesCore provides the core functionalities of Safe Eyes."""
 
 import datetime
 import logging
 import threading
 import time
 
 from safeeyes import utility
-from safeeyes.model import Break
 from safeeyes.model import BreakType
```

Comment:
Very opnionated rule, I prefer such format but would love to hear your thoughts

```python
from package.module import model1, model2, ...

from package.module2 import (
    model3,
    model4,
    ...
)
```

Reply:
Apart from removing an unused import, this PR does not touch the import ordering at all.
There are options for this in ruff - however, for that we need to figure out how to deal with PyGObject's `gi.require_version` calls - those need to be mixed before the respective `import` statements.
Thus, I would also prefer to leave this for a future PR.